study.md

```diff
@@ -18,17 +18,14 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works?
 
 ```md
-<!-- your answer here -->
-```
+The event loop is something that looks at the task queue and the stack and puts the queue onto the stack.```
 
 In your own words, describe the difference between sync and async:
 
 ```md
-<!-- your answer here -->
-```
+Sync code is performed at the same time, meaning it processes tasks one at a time from the stack. async means that more than one process can occur, or 2 processes are relagated to different parsers.```
```

Comment:
Sync code is executed in order. A new function won't run until the previous ones stop. Async allows for multiple things to happen simultaneously and when a response is received, the previous function finishes.